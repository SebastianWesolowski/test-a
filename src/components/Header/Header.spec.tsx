import renderer from "react-test-renderer";
import "@testing-library/jest-dom";

import { Header } from ".";

describe("Components", () => {
  describe("Header", () => {
    const component = renderer.create(<Header />);
    const tree = component.toJSON();

    it("should render without errors", () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
