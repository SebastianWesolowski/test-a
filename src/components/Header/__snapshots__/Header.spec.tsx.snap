// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Header should render without errors 1`] = `
<div
  className="fixed bottom-0 right-0 h-24 w-24 lg:left-0 lg:top-0"
  style={
    Object {
      "backgroundColor": "#fac846",
      "padding": "0px",
      "zIndex": 2,
    }
  }
>
  <svg
    fill="none"
    height="100%"
    style={
      Object {
        "backgroundColor": "#fac846",
      }
    }
    viewBox="0 0 328 213"
    width="100%"
    xmlns="http://www.w3.org/2000/svg"
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <path
      d="M265.52 32.8963L269.569 15.0326L270.394 11.4598C270.456 10.8233 270.611 10.1994 270.853 9.60858C271.003 9.43579 271.268 9.3386 271.587 9.28761V9.31205H273.91L273.954 9.31239H316.484L272.377 203.347C272.349 203.46 272.322 203.574 272.295 203.688H220.913V203.797C220.267 203.775 219.625 203.664 219.005 203.466C218.59 203.202 218.238 202.85 217.972 202.434C217.705 202.019 217.532 201.55 217.464 201.06C216.908 199.219 216.406 197.365 215.904 195.514L215.904 195.511L215.902 195.507C215.578 194.307 215.253 193.108 214.914 191.915L198.695 133.546C197.521 129.307 190.843 128.974 189.852 133.546L177.395 190.823C176.698 194.081 176 197.32 175.267 200.578L175.246 200.669C174.997 201.784 174.759 202.852 173.634 203.429C172.484 204.043 170.706 203.923 169.145 203.817C168.59 203.78 168.06 203.744 167.598 203.744H167.542V203.689H116.155L115.693 201.661L113.255 190.823L110.759 179.845C110.398 177.585 109.783 175.374 108.925 173.255C105.65 166.629 99.0742 166.664 92.8964 166.697C92.4637 166.699 92.0329 166.701 91.6053 166.701H62.0329C60.2897 162.888 56.4669 160.241 52.0314 160.241C45.9518 160.241 41.0233 165.214 41.0233 171.348C41.0233 177.482 45.9518 182.455 52.0314 182.455C56.4741 182.455 60.302 179.8 62.0413 175.976H95.715C95.9642 175.976 96.2301 175.967 96.5048 175.957C97.7269 175.914 99.1244 175.865 100.008 176.605C100.42 176.95 100.709 177.509 100.927 178.147L101.575 180.993C101.597 181.102 101.619 181.207 101.641 181.307L108.081 209.594C108.1 209.678 108.121 209.761 108.143 209.843L108.85 212.945H111.791C112.034 212.981 112.29 213 112.557 213H172.166C174.757 212.998 177.28 212.164 179.371 210.62C181.462 209.076 183.012 206.902 183.798 204.41C184.241 202.918 184.544 201.368 184.846 199.826L184.847 199.819C184.994 199.068 185.14 198.319 185.302 197.579L192.843 162.944L194.81 153.863L205.539 192.359L208.456 202.874C210.144 208.964 214.51 212.944 220.987 213H276.027C276.243 212.996 276.458 212.977 276.67 212.944H279.607L280.229 210.207C280.315 210.015 280.388 209.816 280.448 209.612L295.492 143.066L321.013 30.9344C321.711 27.8065 322.449 24.684 323.188 21.5596L323.189 21.5567L323.189 21.5547C324.391 16.4712 325.593 11.3816 326.627 6.2579V6.09503L328 0.0560422H274.5C273.592 -0.0188502 272.681 -0.0185111 271.777 0.0560422H271.587V0.0728379C270.412 0.183343 269.25 0.419261 268.119 0.778351C264.102 2.3889 262.395 5.83213 261.496 9.77519L259.295 19.9198L256.773 30.7924C251.67 31.7595 247.81 36.279 247.81 41.7084C247.81 47.8427 252.738 52.8156 258.818 52.8156C264.897 52.8156 269.826 47.8427 269.826 41.7084C269.826 38.1187 268.138 34.9267 265.52 32.8963Z"
      fill="black"
    />
    <path
      d="M211.043 27.8799H190.469C188.729 31.7036 184.901 34.3591 180.459 34.3591C174.379 34.3591 169.451 29.3863 169.451 23.2519C169.451 17.1176 174.379 12.1447 180.459 12.1447C184.901 12.1447 188.729 14.8002 190.469 18.6239H207.814C209.976 18.4941 212.146 18.5872 214.29 18.9016C219.757 19.9938 222.344 24.3997 223.885 29.2498L227.28 39.7276L256.946 133.176C258.763 138.859 249.938 141.192 248.103 135.638L214.932 31.1936C214.272 29.1017 213.501 27.8799 211.043 27.8799Z"
      fill="black"
    />
    <path
      d="M117.474 27.8799H107.072C101.164 27.8799 101.164 18.6239 107.072 18.6239H119.07C121.855 18.5651 124.577 19.4669 126.785 21.1802C128.994 22.8934 130.557 25.3156 131.216 28.0465C133.506 37.0896 135.509 46.2316 137.51 55.3625C138.508 59.914 139.505 64.4627 140.536 68.9951L148.939 106.167C150.26 111.98 141.417 114.442 140.096 108.629L125.418 44.1335C125.015 42.3836 124.625 40.6281 124.235 38.8725C123.615 36.0779 122.994 33.2828 122.318 30.5086C121.657 27.8799 119.768 27.8799 117.474 27.8799Z"
      fill="black"
    />
    <path
      d="M61.9608 18.6239H88.7249C94.6326 18.6239 94.6326 27.8799 88.7249 27.8799H62.0113C60.284 31.7334 56.4405 34.4147 51.9763 34.4147C45.8968 34.4147 40.9683 29.4418 40.9683 23.3075C40.9683 17.1731 45.8968 12.2003 51.9763 12.2003C56.3975 12.2003 60.2098 14.8301 61.9608 18.6239Z"
      fill="black"
    />
    <path
      d="M121.877 110.721H88.2295C86.0576 110.743 83.9307 111.348 82.067 112.474C80.2034 113.599 78.6703 115.205 77.6251 117.126C76.8631 118.534 76.1685 119.979 75.4743 121.424L75.4708 121.431C75.0799 122.245 74.6888 123.059 74.286 123.865C74.1211 124.193 73.9475 124.6 73.7592 125.042C73.1305 126.516 72.3389 128.371 71.1671 128.955C70.3253 129.25 69.4275 129.345 68.5435 129.233H66.1693C64.43 125.409 60.6021 122.754 56.1594 122.754C50.0798 122.754 45.1514 127.727 45.1514 133.861C45.1514 139.995 50.0798 144.968 56.1594 144.968C60.5804 144.968 64.3927 142.339 66.1438 138.545C67.355 138.54 68.5599 138.524 69.7544 138.489C71.953 138.471 74.1071 137.861 75.994 136.722C77.8808 135.583 79.432 133.956 80.4872 132.01C81.2576 130.577 81.9647 129.104 82.6709 127.633L82.6717 127.631L82.6731 127.628L82.6748 127.625C83.0437 126.856 83.4123 126.088 83.7896 125.327C83.9559 124.991 84.1252 124.597 84.3055 124.178C85.0199 122.514 85.906 120.45 87.459 120.051C88.5438 119.922 89.6399 119.922 90.7247 120.051H121.767C127.675 120.051 127.675 110.795 121.767 110.795L121.877 110.721Z"
      fill="black"
    />
    <path
      d="M158.076 80.0282L168.368 47.5952L169.854 42.9487C171.689 37.2655 162.809 34.8219 161.011 40.4866L150.7 72.9011L149.232 77.5662C147.398 83.2493 156.278 85.6744 158.076 80.0282Z"
      fill="black"
    />
    <path
      d="M20.9973 82.9982C19.2486 86.7977 15.4332 89.4324 11.008 89.4324C4.92847 89.4324 0 84.4595 0 78.3251C0 72.1908 4.92847 67.2179 11.008 67.2179C15.465 67.2179 19.3034 69.8906 21.0347 73.7342H22.1629C22.6126 73.7342 23.0704 73.7464 23.5309 73.7586C24.4604 73.7834 25.4015 73.8084 26.3092 73.7342C27.4972 73.5341 28.5588 72.8688 29.2631 71.883L37.6842 63.386C38.2924 62.7861 38.8892 62.1449 39.4928 61.4964C41.3108 59.543 43.1904 57.5234 45.6284 56.3699C48.3674 55.074 51.3674 55.1213 54.3202 55.1679C54.8681 55.1765 55.4148 55.1851 55.9576 55.1851H68.8003C69.0883 55.1851 69.3757 55.1839 69.6625 55.1828C73.1211 55.1684 76.4822 55.1544 79.5148 57.24C83.1842 59.7391 84.2483 63.46 85.1656 67.4956L89.6606 87.2109L89.6239 87.1554C90.9449 92.9682 82.1017 95.4302 80.7808 89.6175L75.8822 68.3472C75.4052 66.2368 74.9648 64.3856 72.488 64.3856H52.1048C51.154 64.2551 50.186 64.4029 49.3161 64.8114C48.6333 65.3521 48.0173 65.9737 47.4814 66.6626L39.1336 75.0855C38.6454 75.5782 38.1563 76.0989 37.6603 76.6271C35.8074 78.5998 33.8564 80.6771 31.483 81.7869C28.5991 83.158 25.2803 83.0873 22.089 83.0193C21.7232 83.0115 21.359 83.0038 20.9973 82.9982Z"
      fill="black"
    />
  </svg>
</div>
`;
